src/libcollections/string.rs

```diff
@@ -343,7 +343,8 @@ impl String {
     /// This function is unsafe because we dereference memory until we find the
     /// NUL character, which is not guaranteed to be present. Additionally, the
     /// slice is not checked to see whether it contains valid UTF-8
-    #[unstable = "just renamed from `mod raw`"]
+    #[deprecated = "use ffi::c_str_to_bytes + str::from_utf8 + .to_string()"]
```

Comment:
I think we should offer this convenience in `ffi`, something like `ffi::c_str_to_string`.